### PR TITLE
style: session slides upload page

### PR DESCRIPTION
**Pull request by Prerna290**
## Type
- ( ) Fix
- ( ) Refactor
- (X) Style
- ( ) Docs
- ( ) Feat
- ( ) Hotfix

### CODE REVIEW

File: apps/commudle-admin/src/app/app-shared-components/featured-projects/featured-projects.component.ts [hunk 1]
```diff
@@ -2,20 +2,43 @@ import { Component, Input, OnInit } from '@angular/core';
 import { FeaturedItemsService } from 'apps/commudle-admin/src/app/services/featured-items.service';
 import { IFeaturedItems } from 'apps/shared-models/featured-items.model';
 import { staticAssets } from 'apps/commudle-admin/src/assets/static-assets';
+import { CommonModule } from '@angular/common';
+import { NbCardModule } from '@commudle/theme';
+import { SharedComponentsModule } from 'apps/shared-components/shared-components.module';
+import { FeaturedProjectsCardComponent } from '../featured-projects-card/featured-projects-card.component';
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/app-shared-components/featured-projects/featured-projects.component.ts]: let's fix the path of import!

File: apps/commudle-admin/src/app/app-shared-components/featured-projects/featured-projects.component.ts [hunk 2]
```diff
@@ -2,20 +2,43 @@ import { Component, Input, OnInit } from '@angular/core';
...
+import { SkeletonCardsComponent } from '../../feature-modules/skeleton-screens/components/skeleton-cards/skeleton-cards.component';
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/app-shared-components/featured-projects/featured-projects.component.ts]: let's fix the path of import!

File: apps/commudle-admin/src/app/app-shared-components/labs-featured/labs-featured.component.html [hunk 3]
```diff
@@ -6,9 +6,12 @@
...
   <nb-card-body>
     <div class="heading">
-      <button nbButton status="basic" class="image">
+      <button *ngIf="showStarsEmojiOnHeading" nbButton status="basic" class="image">
         <img alt="Experts" src="{{ staticAssets.emojis.stars }}" title="Experts" />
       </button>
+      <button *ngIf="showFlaskEmojiOnHeading" nbButton class="emoji" size="small">
+        <fa-icon [icon]="faFlask"></fa-icon>
+      </button>
       <div class="text">
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/app-shared-components/labs-featured/labs-featured.component.html:15]: let's remove button and make it depended

File: apps/commudle-admin/src/app/app.module.ts [hunk 4]
```diff
@@ -161,6 +164,46 @@ export function initApp(appInitService: AppInitService): () => Promise<any> {
     FillDataFormPaidComponent,
     CheckFillDataFormComponent,
   ],
+  providers: [
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/app.module.ts]: let's fix this file

File: apps/commudle-admin/src/app/components/speaker-resource-form/speaker-resource-form.component.ts [hunk 5]
```diff
@@ -33,23 +45,34 @@ export class SpeakerResourceFormComponent implements OnInit {
     private sanitizer: DomSanitizer,
     private communitiesService: CommunitiesService,
     private toastLogService: LibToastLogService,
+    private nbToastrService: NbToastrService,
     private router: Router,
+    private authWatchService: LibAuthwatchService,
+    private appUsersService: AppUsersService,
+    private cdr: ChangeDetectorRef,
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/components/speaker-resource-form/speaker-resource-form.component.ts]: let's remove if not required

File: apps/commudle-admin/src/app/components/speaker-resource-form/speaker-resource-form.component.ts [hunk 6]
```diff
@@ -80,14 +106,53 @@ export class SpeakerResourceFormComponent implements OnInit {
 
   submitForm() {
     this.speakerResourcesService
-      .createOrUpdateByToken(this.token, this.speakerResourceForm.value, this.eventId)
+      .createOrUpdateByToken(this.token, this.getPdfFormData(), this.eventId)
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/components/speaker-resource-form/speaker-resource-form.component.ts]: let's change function name -> getFormData()

File: apps/commudle-admin/src/app/feature-modules/community-builds/community-builds.module.ts [hunk 7]
```diff
@@ -42,6 +41,7 @@ import { ExploreExpertsComponent } from './components/community-builds/explore-e
 import { BuildsComponent } from './components/community-builds/builds/builds.component';
 import { SkeletonCardsComponent } from 'apps/commudle-admin/src/app/feature-modules/skeleton-screens/components/skeleton-cards/skeleton-cards.component';
 import { ListingPagesLayoutComponent } from 'apps/commudle-admin/src/app/app-shared-components/listing-pages-layout/listing-pages-layout.component';
+import { FeaturedProjectsComponent } from '../../app-shared-components/featured-projects/featured-projects.component';
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/community-builds/community-builds.module.ts]: let's fix the path of import!

File: apps/commudle-admin/src/app/feature-modules/homepage/components/homepage-dashboard/homepage-dashboard.component.ts [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+@Component({
+  selector: 'commudle-homepage-dashboard',
+  templateUrl: './homepage-dashboard.component.html',
+  styleUrls: ['./homepage-dashboard.component.scss'],
+})
+export class HomepageDashboardComponent implements OnInit {
+  currentUser: ICurrentUser;
+  staticAssets = staticAssets;
+  userProfileDetails;
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/homepage/components/homepage-dashboard/homepage-dashboard.component.ts:15]: let's define datatype

File: apps/commudle-admin/src/app/feature-modules/homepage/components/homepage/homepage.component.ts [hunk 9]
```diff
@@ -12,7 +12,7 @@ import { FooterService } from 'apps/commudle-admin/src/app/services/footer.servi
   templateUrl: './homepage.component.html',
   styleUrls: ['./homepage.component.scss'],
 })
-export class HomepageComponent implements OnInit, OnDestroy, AfterViewInit {
+export class HomepageComponent implements OnInit, OnDestroy {
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/homepage/components/homepage/homepage.component.ts]: why we remove AfterViewInit

File: apps/commudle-admin/src/app/feature-modules/speaker-resources/components/speaker-resource/speaker-resource.component.ts [hunk 10]
```diff
@@ -51,6 +53,7 @@ export class SpeakerResourceComponent implements OnInit, OnDestroy {
     this.subscriptions.push(
       this.speakerResourcesService.getDetails(speakerId).subscribe((data) => {
         this.speakerResource = data;
+        console.log(this.speakerResource, 'speaker-resource');
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/speaker-resources/components/speaker-resource/speaker-resource.component.ts]: let's remove console

File: apps/commudle-admin/src/app/feature-modules/speaker-resources/components/speaker-resource/speaker-resource.component.ts [hunk 11]
```diff
@@ -25,6 +26,7 @@ export class SpeakerResourceComponent implements OnInit, OnDestroy {
 
   subscriptions: Subscription[] = [];
   iframe;
+  ERegistrationType = ERegistrationType;
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/speaker-resources/components/speaker-resource/speaker-resource.component.ts]: why we are using this?

File: apps/commudle-admin/src/app/services/app-users.service.ts [hunk 12]
```diff
@@ -174,4 +174,8 @@ export class AppUsersService {
       delete_profile: deleteProfile,
     });
   }
+
+  getProfileStats(): Observable<IUser> {
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/services/app-users.service.ts:178]: why we are using IUser model ?

File: apps/commudle-admin/src/assets/static-assets.ts [hunk 13]
```diff
@@ -44,6 +44,9 @@ export const staticAssets = {
 
     medal_of_honour:
       'https://json.commudle.com/rails/active_storage/blobs/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOGNxQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c32f54e069ae256b46224c8d591a14e100abcc9d/b3d37220175f46299a2639f1bf15ded2.png',
+
+    smily:
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/assets/static-assets.ts]: let's correct the spelling

File: libs/shared/environments/src/lib/environments.ts [hunk 14]
```diff
@@ -72,6 +72,6 @@ const environments: { [type: string]: Environment } = {
 };
 
 // export const environment = environments['local'];
-// export const environment = environments['test'];
+export const environment = environments['test'];
 // export const environment = environments['staging'];
-export const environment = environments['production'];
+// export const environment = environments['production'];
```
Comment (ArshdeepGrover) [inline libs/shared/environments/src/lib/environments.ts]: let's fix this profile

**Review by ArshdeepGrover — CHANGES_REQUESTED** [review]

File: apps/commudle-admin/src/app/app-shared-components/featured-projects/featured-projects.component.ts (hunk 1 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/app-shared-components/featured-projects/featured-projects.component.ts]: Done

File: apps/commudle-admin/src/app/components/speaker-resource-form/speaker-resource-form.component.ts [hunk 15]
```diff
@@ -1,13 +1,18 @@
-import { Component, OnInit, TemplateRef, ViewChild } from '@angular/core';
+import { ChangeDetectorRef, Component, OnInit, TemplateRef, ViewChild } from '@angular/core';
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/components/speaker-resource-form/speaker-resource-form.component.ts:1]: let's remove if not required

**Review by ArshdeepGrover — APPROVED** [review]

**Pull request merged**